2/contrib/openshift/base-plugins.txt

```diff
@@ -29,9 +31,22 @@ openshift-client:1.0.35
 openshift-login:1.0.26
 openshift-sync:1.0.46
 pam-auth:1.6
+parameterized-trigger:2.36
+pipeline-build-step:2.12
+pipeline-github-lib:1.0
+pipeline-input-step:2.11
+pipeline-model-api:1.7.2
+pipeline-model-definition:1.6.0
+pipeline-utility-steps:2.5.0
 prometheus:2.0.0
-snakeyaml-api:1.27.0
 script-security:1.77
+snakeyaml-api:1.27.0
 ssh-credentials:1.18.1
 subversion:2.13.2
```

Comment:
so you want to bump subversion in a separate PR @adambkaplan ?

Reply:
yes - there are other issues with the 3.11 plugins which need to be addressed.

Reply:
Changing my decision here - I am going to backport the subversion fix here, but other plugin updates will be held back due to existing issues with the 1.0.50 openshift-sync plugin